src/containers/toputilizers/TopUtilizersReducer.js

```diff
@@ -63,6 +64,9 @@ export default function reducer(state :Immutable.Map<*, *> = INITIAL_STATE, acti
     case actionTypes.SUBMIT_TOP_UTILIZERS_FAILURE:
       return state.set('isGettingResults', false);
 
+    case actionTypes.GET_TOP_UTILIZERS_NEIGHBORS_SUCCESS:
+      return state.set('neighbors', action.neighbors);
```

Comment:
Use `Immutable.fromJS(action.neighbors)` instead in order to store the data as an immutable object.